cmd/build.sh

```diff
@@ -154,14 +154,17 @@ do
     echo -en "\t - $goos/$goarch..."
 
     mi_archive_name="wso2$filename-cli-$build_version-$pos-$parch"
```

Comment:
Shall we use `mi_archive_name="$mi_dir_name-$pos-$parch" instead?

Reply:
Fixed in cd9c85e